### ISSUE TITLE
Non intuitive error when -o is not passed

### ISSUE THREAD

**Issue by clementval** [claw-project/claw-compiler#498]
When the output file is not provided an error is raised instead of printing the result on the stdout. 

```bash
../../../driver/clawfc_test: line 356: : No such file or directory
```
